test/hotspot/jtreg/vmTestbase/nsk/jdb/caught_exception/caught_exception002/caught_exception002.java

```diff
@@ -112,11 +108,10 @@ protected void runCases() {
         jdb.contToExit(1);
     }
 
-    private void checkCatch (String[] reply, int i) {
+    private void checkCatch(String[] reply, int i) {
         Paragrep grep;
         int count;
-        String found;
-        Vector v = new Vector();
+        Vector<String> v = new Vector<>();
```

Comment:
As long as we cleaning up the coding here, the declarations can be moved to the first use? Here... and everywhere.